app/src/main/res/values/colors.xml

```diff
@@ -6,6 +6,7 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="red">#FF0000</color>
```

Comment:
Since all other colors are in RGBA it'ld be nice to do the same for this one.

Reply:
Done